modules/exploits/linux/http/control_web_panel_api_cmd_exec.rb

```diff
@@ -0,0 +1,132 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  prepend Msf::Exploit::Remote::AutoCheck
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::CmdStager
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Control Web Panel /admin/index.php Unauthenticated RCE',
+        'Description' => %q{
+          Control Web Panel (CWP) versions <= 0.9.8.1208 are vulnerable to
+          unauthenticated OS command injection. User input passed via the
+          "key" GET parameter to /admin/index.php (when the "api" parameter is set)
+          is not properly sanitized before being used to execute OS commands.
+          This can be exploited by unauthenticated attackers to inject and execute
+          arbitrary OS commands with the privileges of the root user on the web server.
+
+          Successful exploitation usually requires "Softaculous" and/or "SitePad"
+          to be installed through the Scripts Manager.
+        },
+        'Author' => [
+          'Lukas Johannes Möller', # Metasploit module
+          'Egidio Romano' # Vulnerability discovery
+        ],
+        'References' => [
+          ['CVE', '2025-67888'],
+          ['URL', 'https://karmainsecurity.com/KIS-2025-09'],
+          ['URL', 'https://www.cve.org/CVERecord?id=CVE-2025-67888'],
+          ['URL', 'https://control-webpanel.com']
+        ],
+        'DisclosureDate' => '2025-12-16',
+        'License' => MSF_LICENSE,
+        'Platform' => ['unix', 'linux'],
```

Comment:
I don't think it's working on Unices that aren't Linux.

```suggestion
        'Platform' => ['linux'],
```

Reply:
Yes, you are right.